components/EditDeleteInfoForm/style.js

```diff
@@ -92,3 +92,57 @@ export const EntryDeleteButton = styled.button`
   height: 40px;
   width: 40px;
 `;
+
+export const ModalHeadline = styled.h2`
+  text-transform: uppercase;
+  font-size: 1.3rem;
+  font-weight: 700;
+  color: #ff0000;
+  margin: 0px 0px 10px 0px;
+`;
+
+export const ModalMessage = styled.p`
+  font-size: 1rem;
+  font-weight: 500;
+  color: #000000;
+  margin-top: 0px;
+`;
+
+export const ModalDeleteButton = styled.button`
```

Comment:
You might want to consider having as base style for your buttons. And then creating specific button styles based off of that. 

[See example here](https://styled-components.com/docs/basics#extending-styles).

The following code seems to be shared by both the `ModalDeleteButton` and the `ModalKeepButton`

``` css
text-transform: uppercase;
color: #ffffff;
height: 25px;
width: 120px;
border-radius: 5px;
border: none;

&:focus {
  font-weight: 600;
}
```

Having one base button will make maintaining things easier in the future.